website/src/components/genspectrum/GsMutationFilter.tsx

```diff
@@ -9,13 +9,21 @@ export type MutationFilter = {
     aminoAcidInsertions: string[];
 };
 
+export type MutationType =
+    | 'nucleotideMutations'
+    | 'nucleotideInsertions'
+    | 'aminoAcidMutations'
+    | 'aminoAcidInsertions';
```

Comment:
Actually we should also export those in the components rather than duplicating them here. I'd propose to export them as values and as a type:

```
export const mutationType = {
    nucleotideMutations: 'nucleotideMutations',
    ...
}
```

Then use them in the Zod type and also export the Zod type from the components.
(While we're there, we can also do #983 ;) )